x-pack/plugins/osquery/public/editor/osquery_highlight_rules.ts

```diff
@@ -81,9 +79,9 @@ const builtinFunctions = [
   'ifnull',
   'isnull',
   'nvl',
-].join('|');
+];
 
-const dataTypes = [
+export const dataTypes = [
```

Comment:
it looks like these 5 constants above don't need to be exported

(sorry about these comments, if your PR is really WIP :) )

Reply:
Oh don't worry, I forgot about it, previously it was meant to be exported, but in the end I put the logic in the very same file.

Reply:
Thanks for catching this 👍